pandas/tests/arrays/sparse/test_accessor.py

```diff
@@ -119,3 +119,11 @@ def test_series_from_coo(self, dtype, dense_index):
             )
 
         tm.assert_series_equal(result, expected)
+
+    @td.skip_if_no_scipy
+    def test_series_from_coo_incorrect_format(self):
```

Comment:
```suggestion
    def test_series_from_coo_incorrect_format_raises(self):
```

Reply:
I like that. added type error to the end to be more specific.

Reply:
having _raises at the end of the test name is for consistency. if you can lose the type error bit that'll be great

Reply:
will do!